HW_1/task8.py

```diff
@@ -0,0 +1,16 @@
+#Задача 8: Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, если разрешается сделать 
+#один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+#*Пример:*
+#3 2 4 -> yes
+#3 2 1 -> no
+
+print ('Введите размер шоколадки n')
+n = int(input())
```

Comment:
Зачем использовать функцию `print` для вывода сообщения о вводе? Если можно это сообщение, поместите напрямую в внутрь функции `input`.
```suggestion
n = int(input('Введите размер шоколадки n '))
```
И так везде